### ISSUE TITLE
Support for Passing Data among Argo Jobs

### ISSUE THREAD

**Issue by kaushikb11** [onepanelio/core#612]
#### Description

Let’s consider I have triggered an Argo Workflow, it is a DAG of multiple Jobs. Jobs A and B are running parallelly & we want data to pass from Job A to Job B. Example: Job B does Tracking & Job A does OD with Bounding Boxes as outputs. Right now, we pass data through queues and let the Jobs polls on it.

**Comment by rushtehrani** [comment]
Related Argo issue: https://github.com/argoproj/argo/issues/2729

**Comment by sarabala1979** [comment]
@rushtehrani  are you using Argo workflow? would you like to add `onepanel` in argo users.md?

**Comment by sarabala1979** [comment]
I would like to here the more details for the Stream workflow usecase.   Basically, I am waiting for real-world usecases to start the workflow.

**Comment by rushtehrani** [comment]
> are you using Argo workflow? would you like to add `onepanel` in argo users.md?

@sarabala1979 Yes we are and I will absolutely send a PR to add us to users.md.

> I would like to here the more details for the Stream workflow usecase. Basically, I am waiting for real-world usecases to start the workflow.

@kaushikb11 has a great use case with videos. It's basically a combination of this issue and https://github.com/onepanelio/core/issues/613.

I am also exploring other related use cases and will provide more information in https://github.com/argoproj/argo/issues/2729 based on my findings.
